addons/mail/static/src/core/web/follower_subtype_dialog.js

```diff
@@ -60,7 +63,7 @@ export class FollowerSubtypeDialog extends Component {
                     subtype_ids: selectedSubtypes.map((subtype) => subtype.id),
                 }
             );
-            if (!selectedSubtypes.some((subtype) => subtype.id === this.store.mt_comment_id)) {
+            if (!selectedSubtypes.some((subtype) => subtype.eq(this.store.mt_comment))) {
```

Comment:
not sure if working but this looks like `this.store.mt_comment.notIn(selectedSubtypes)`